<launch>
  <arg name="camera_config" default="$(find dbot_ros)/config/camera.yaml"/>
  <arg name="filter_config" default="$(find dbot_ros)/config/particle_tracker.yaml"/>
  <arg name="service_config" default="$(find dbot_ros)/config/object_tracker_services.yaml"/>

  <node
    name="object_tracker_service"
    type="object_tracker_service"
    pkg="dbot_ros"
    output="screen"
    respawn="false"
    launch-prefix="xterm -bg FloralWhite -fg MistyRose4 -e">
        <rosparam command="load" file="$(arg filter_config)" />
        <rosparam command="load" file="$(arg camera_config)" />
        <rosparam command="load" file="$(arg service_config)" />
  </node>
</launch>
